scene/2d/camera_2d.cpp

```diff
@@ -527,11 +527,14 @@ void Camera2D::align() {
 
 void Camera2D::set_follow_smoothing(float p_speed) {
 
+	// enable / disable should now be called explicitly except using _set_old_smoothing
+	// for backward compatibility
 	smoothing = p_speed;
-	if (smoothing > 0 && !(is_inside_tree() && Engine::get_singleton()->is_editor_hint()))
-		set_process_internal(true);
-	else
-		set_process_internal(false);
+
+	//	if (smoothing > 0 && !(is_inside_tree() && Engine::get_singleton()->is_editor_hint()))
+	//		set_process_internal(true);
+	//	else
+	//		set_process_internal(false);
 }
```

Comment:
This is the most important bit to look at. Do we want to preserve this on / off functionality (and use the same logic as `_set_old_smoothing`) or deprecate it now? We could keep it the same in 3.2 and deprecate in 4.0, but I'm not clear how old this functionality is - maybe we should deprecate in 3.2.

Reply:
I would remove this, and remove `_set_old_smoothing`.
It's bound with an underscore so it's not part of the documented "public" API even though users can use it. If so do, they've had it coming for years that they're using something which is apparently not meant to be used (hidden and named "old").

Reply:
```
29ea3992342 (Juan Linietsky     2015-12-12 09:26:13 -0300 555) void Camera2D::_set_old_smoothing(float p_val) {
29ea3992342 (Juan Linietsky     2015-12-12 09:26:13 -0300 556)  //compatibility
```

See 29ea3992342, it seems @reduz just added this for compat but there's no clear indication that anyone actually knew about it and used it.

Reply:
Ok removed them both.